pyzabbix/sender.py

```diff
@@ -131,6 +131,70 @@ def __repr__(self):
         return result
 
 
+class ZabbixActiveChecksResponse(object):
+    """The :class:`ZabbixActiveChecksResponse`
+    contains the parsed response from Zabbix.
+    """
+    def __init__(self):
+        self.checks = None
+
+    def __repr__(self):
+        """Represent detailed ZabbixActiveChecksResponse view."""
+        if self.checks is None:
+            return ''
+        else:
+            return "[%s]" % ", ".join([ch.__repr__() for ch in self.checks])
+
+    def parse(self, response):
+        """Parse zabbix response for active checks."""
+        self.checks = []
+        data = response.get('data')
+        for o in data:
+            # lastlogsize is optional in 2.2; required in 2.4, 3.0, 3.2, 3.4
+            # mtime       is unused   in 3.2; required in 2.4, 3.0, 3.2, 3.4
+            ch = ZabbixCheck(
+                o.get('key'),
```

Comment:
If some keys are required and not optional, we should get them explicitly to raise exception on error. Otherwise we may get empty checks on error.